components/assetcard/package.json

```diff
@@ -20,15 +20,15 @@
   "peerDependencies": {
     "@spectrum-css/checkbox": ">=7",
     "@spectrum-css/icon": ">=3",
-    "@spectrum-css/typography": ">=4 <=5",
-    "@spectrum-css/vars": ">=9"
+    "@spectrum-css/tokens": ">=10",
+    "@spectrum-css/typography": ">=4 <=5"
```

Comment:
Are we still using typography here?  It seems like we're hooking into the variables directly and not using the classes from that package anymore.